components/camel-google-sheets/pom.xml

```diff
@@ -40,8 +40,55 @@
     <camel.osgi.private.pkg>org.apache.camel.component.google.sheets.internal</camel.osgi.private.pkg>
     <camel.osgi.export.pkg>org.apache.camel.component.google.sheets</camel.osgi.export.pkg>
     <camel.osgi.export.service>org.apache.camel.spi.ComponentResolver;component=google-sheets</camel.osgi.export.service>
+
+    <citrus.version>2.8.0</citrus.version>
```

Comment:
maybe these placeholders can be in the parent/pom.xml, for the future we can use them directly and have a common version in all components

Reply:
very true! can I force push the change to this PR?

Reply:
Sure!

Reply:
Done!